conf/application.conf

```diff
@@ -104,9 +104,9 @@ contact-frontend {
 }
 
 timeoutDialog {
-    enabled = true
-    timeout = 20
-    timeoutCountdown = 10
+  enabled = true
+  timeout = 10
+  timeoutCountdown = 3
```

Comment:
Need to check but I think the prompt should trigger at 2 minutes before sign-out